homeassistant/components/water_heater/wink.py

```diff
@@ -0,0 +1,136 @@
+"""
+Support for Wink water heaters.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/water_heater.wink/
+"""
+import logging
+
+from homeassistant.components.water_heater import (
+    ATTR_TEMPERATURE, STATE_ECO, STATE_ELECTRIC,
+    STATE_PERFORMANCE, SUPPORT_AWAY_MODE, STATE_HEAT_PUMP,
+    STATE_GAS, STATE_HIGH_DEMAND,
+    SUPPORT_OPERATION_MODE, SUPPORT_TARGET_TEMPERATURE,
+    WaterHeaterDevice)
+from homeassistant.components.wink import DOMAIN, WinkDevice
```

Comment:
Import the wink domain as another name. The domain of this platform is water_heater.